test-environment.md

```diff
@@ -1,73 +1,59 @@
 <!-- START_METADATA
 ---
-title: Vipps Merchant Test (MT)
+title: The Vipps and MobilePay testing environment
+sidebar_label: Testing environment
 sidebar_position: 3
 pagination_next: null
 pagination_prev: null
 ---
 END_METADATA -->
 
-# The Vipps Merchant Test (MT)
-
-The Vipps MT is available for all Vipps merchants who have API access.
+# The testing environment
 
 <!-- START_COMMENT -->
 
 ℹ️ Please use the website:
-[Vipps MobilePay Technical Documentation](https://vippsas.github.io/vipps-developer-docs/).
-
-## Table of contents
-
-* [About the test environment](#about-the-test-environment)
-  * [Limitations of the test environment](#limitations-of-the-test-environment)
-* [Access to the test environment](#access-to-the-test-environment)
-  * [Merchants](#merchants)
-  * [Partners](#partners)
-* [Test server](#test-server)
-* [Test merchants](#test-merchants)
-* [Test users](#test-users)
-  * [Phone number](#phone-number)
-  * [Payment card](#payment-card)
-  * [Email address](#email-address)
-  * [Other data](#other-data)
-* [Test amounts](#test-amounts)
-* [Vipps test apps](#vipps-test-apps)
-  * [Install Vipps MT on iOS](#install-vipps-mt-on-ios)
-  * [Install Vipps MT on Android](#install-vipps-mt-on-android)
-  * [Other versions](#other-versions)
-  * [Limitations of the test apps](#limitations-of-the-test-environment)
+[Vipps MobilePay Technical Documentation](https://vippsas.github.io/vipps-developer-docs/docs/vipps-developers/test-environment).
 
 <!-- END_COMMENT -->
 
-## About the test environment
+## Testing with MobilePay
+
+Vipps and MobilePay have recently merged so the information is still under development and will be updated soon.
+
+You can find MobilePay testing information under the _Development Guide > Testing_ section for each API in the
+[MobilePay developers website](https://developer.mobilepay.dk/).
```

Comment:
Maybe skip the s?
```suggestion
[MobilePay developer website](https://developer.mobilepay.dk/).
```

Reply:
I'm going to revert changes to this whole page, based on Henriette's comment